.travis.yml

```diff
@@ -1,15 +1,24 @@
-sudo: true
-dist: trusty
+# Build matrix / environment variable are explained on:
+# http://about.travis-ci.org/docs/user/build-configuration/
+# This file can be validated on:
+# http://lint.travis-ci.org/
+sudo: required
+
 language: ruby
+
 rvm:
   - 2.3
 
 services:
   - docker
 
+env:
+  - TERRAFORM_VERSION=0.11.1 IMAGE_NAME=azure-loadbalancer-module
+
 jobs:
   include:
     - stage: rake build
       install: true
       script:
-      - docker run -v $PWD:/tf-test/module --rm microsoft/terraform-test rake -f ../Rakefile build
\ No newline at end of file
+      - docker build -t ${IMAGE_NAME} .
```

Comment:
I would recommend reverting to this structure instead: https://github.com/Azure/terraform-azurerm-computegroup/blob/master/.travis.yml#L10-L15

Reply:
We want to validate the Dockerfile in the CI.